api/client/client.go

```diff
@@ -312,6 +315,9 @@ func (c *Client) DownloadFile(hash, path, dest, credentials string) error {
 	}
 
 	uri := c.Gateway + "/bzz:/" + hash + "/" + path
+	if publisher != "" {
```

Comment:
please dont do it manually. there is proper api to add query params

```
req.URL.Query().Add("publisher",publisher)
```

Reply:
Good point, fixed